requirements.txt

```diff
@@ -1,2 +1,3 @@
 mako
 PyYAML
+nose
```

Comment:
Some upstream keep a ``test_requirements.txt`` to keep the requirements that are only for tests out of the requirements that are for running the app.

You may also just add a comment that this is needed only to run the tests